homeassistant/components/axis.py

```diff
@@ -0,0 +1,346 @@
+"""
+Support for Axis devices.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/axis/
+"""
+
+import json
+import logging
+import os
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.const import (ATTR_LOCATION, ATTR_TRIPPED,
+                                 CONF_HOST, CONF_INCLUDE, CONF_NAME,
+                                 CONF_PASSWORD, CONF_TRIGGER_TIME,
+                                 CONF_USERNAME, EVENT_HOMEASSISTANT_STOP)
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers import discovery
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import track_point_in_utc_time
+from homeassistant.loader import get_component
+from homeassistant.util.dt import utcnow
+
+
+REQUIREMENTS = ['http://github.com/Kane610/axis'
+                '/archive/v5.zip'
+                '#axis==5']
```

Comment:
Please publish your library to pypi rather than use a github path if you can.  [This](http://peterdowns.com/posts/first-time-with-pypi.html) is a good tutorial if you're not sure how to do it.

If you need to keep it on github for some reason please switch to ```https```.